src/main/java/com/solvd/pages/CartPage.java

```diff
@@ -0,0 +1,27 @@
+package com.solvd.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class CartPage extends AbstractPage{
+    @FindBy(xpath = "//div[@class='container-fluid cart-info product-list']/table/tbody/tr[position() > 1]/td[2]/a")
```

Comment:
is not good locator, try this one //td[contains(@class,'align_left')]/a
It targets a elements within td elements that contain the class align_left, making it flexible and likely to capture all relevant product links.